### DISCUSSION TITLE
PEST_GSA question

### DISCUSSION THREAD

**Discussion by jbensabat** [usgs/pestpp#318] [General]
Hello 
I have a simple question regarding the output of PEST_GSA. using the Sobol option, there is one file with extension *.sbl 
the title is "Sobol Sensitivity for PHI"
the question is what is PHI - the objective function?
This is what I guess but it's not written anywhere
best 
jac

**Comment by jtwhite79** [comment]
correct - phi is the sum of the squared weighted residuals.

**Reply by jbensabat** [reply]
Thanks
